vision/google/cloud/vision/image.py

```diff
@@ -67,7 +67,7 @@ def as_dict(self):
         """
         if self.content:
             return {
-                'content': self.content
+                'content': _bytes_to_unicode(b64encode(self.content))
```

Comment:
Yup same as above. HTTP needs the content to be base64 encoded and the PB version needs the content not to be.